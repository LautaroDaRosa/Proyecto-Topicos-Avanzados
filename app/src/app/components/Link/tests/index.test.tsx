import * as React from 'react';

import { Link } from '../index';
import renderWithWrappers from 'utils/render';
import { screen } from '@testing-library/react';

describe('Link', () => {
  it('should render correctly', () => {
    const { container } = renderWithWrappers(
      <Link to="/test">HeaderLink</Link>,
    );
    expect(container).toMatchSnapshot();
    const link = screen.getByRole('link', { name: 'HeaderLink' });
    expect(link).toHaveAttribute('href', '/test');
  });
});
