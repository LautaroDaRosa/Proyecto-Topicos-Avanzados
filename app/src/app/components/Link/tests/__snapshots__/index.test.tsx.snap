// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link should render correctly 1`] = `
.c0 {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
  opacity: 0.8;
}

.c0:active {
  opacity: 0.4;
}

<div>
  <a
    class="c0"
    href="/test"
  >
    HeaderLink
  </a>
  ,
</div>
`;
